client/src/components/Navbar.js

```diff
@@ -38,11 +40,25 @@ export default class Navbar extends Component {
             )
         })
     }
+
+    handleOffsetChange() {
+        window.location.reload()
+    }
+
     render() {
-        const { navItems, activeItem } = this.props
+        const { navItems, activeItem, role } = this.props
         return (
-            <Menu>
+            <Menu stackable>
                 {this.renderMenuItems(navItems, activeItem)}
+                
+                {this.props.timezoneActive &&
+                    <TimeZoneDropdown
```

Comment:
I put the timezone dropdown in the navbar - this way they should be able to notice pretty quickly if their timezone is wrong